document/core/exec/instructions.rst

```diff
@@ -693,50 +693,56 @@ Table Instructions
 
 .. _exec-table.copy:
 
-:math:`\TABLECOPY`
-..................
-
-.. todo:: TODO: multi tables
+:math:`\TABLECOPY~x~y`
+......................
 
 1. Let :math:`F` be the :ref:`current <exec-notation-textual>` :ref:`frame <syntax-frame>`.
 
-2. Assert: due to :ref:`validation <valid-table.copy>`, :math:`F.\AMODULE.\MITABLES[0]` exists.
+2. Assert: due to :ref:`validation <valid-table.copy>`, :math:`F.\AMODULE.\MITABLES[x]` exists.
 
-3. Let :math:`\X{ta}` be the :ref:`table address <syntax-tableaddr>` :math:`F.\AMODULE.\MITABLES[0]`.
+3. Let :math:`\X{ta}_d` be the :ref:`table address <syntax-tableaddr>` :math:`F.\AMODULE.\MITABLES[x]`.
```

Comment:
Used that at first, but since almost everything is expressed in terms of s and d from here on, it seemed clearer to focus on the relation to those names. Changed back.